convert_to_multi_instance.sh

```diff
@@ -335,6 +339,27 @@ function refresh_zen(){
     success "Reconcile loop initiated for Zenservice instances"
 }
 
+function cleanupCSOperators(){
+    title "Checking subs of Common Service Operator in Cloudpak Namespaces"
+    msg "-----------------------------------------------------------------------"
+    for namespace in $requestedNS
+    do
+        # remove cs namespace from zen service cr
+        return_value=$(${OC} get sub -n ${namespace} | (grep ibm-common-service-operator || echo "fail"))
+        if [[ $return_value != "fail" ]]; then
+            local sub=$(${OC} get sub -n ${namespace} | grep ibm-common-service-operator | awk '{print $1}')
+            ${OC} get sub ${sub} -n ${namespace} -o yaml > tmp.yaml 
+            ${YQ} '.spec.source = "'${catalog_source}'"' tmp.yaml || error "Could not replace catalog source for CS operator in namespace ${namespace}"
+            ${OC} apply -f tmp.yaml
+            rm tmp.yaml -f
+            info "Common Service Operator Subscription in namespace ${namespace} updated to use catalog source ${catalog_source}"
+        else
+            info "No Common Service Operator in namespace ${namespace}. Moving on..."
+        fi
+        return_value=""
+    done
```

Comment:
```suggestion
    done
    rm tmp.yaml -f
```

Small optimization. No point in removing the tmp.yaml in the loop because each iteration will overwrite the file anyways